apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -10,6 +10,34 @@
 
 namespace A8C\FSE\Coming_soon;
 
+$blog_locale = function_exists( 'get_blog_lang_code' ) ? get_blog_lang_code() : get_locale();
```

Comment:
it looks like `$blog_locale` is coming back as empty '', however, when the same logic is used on [Line 30](https://github.com/Automattic/wp-calypso/pull/46441/files#diff-403aaf6649d463dcf3cd90c4069b8a9ec7383408b291e8d708335b7fb1a3f636R30) to set $locale, 'en' is returned as expected.

Reply:
Thanks for testing!

I'll check this. Maybe this needs to live in its own method and called on runtime.